Cargo.toml

```diff
@@ -18,5 +18,28 @@ regex = "^0.2"
 version = "^0.0"
 optional = true
 
+[dependencies.serde]
+version = "^1"
+optional = true
+
+[dependencies.serde_derive]
+version = "^1"
+optional = true
+
+[dev-dependencies]
+serde        = "^1"
```

Comment:
`serde` and `serde_derive` are already a dependencies, it’s not necessary to duplicated them here.